### PR TITLE
NMS-7667: Unclear color behavior in surveillance view

### PR DESCRIPTION
**Pull request by indigo423**
Added documentation to describe coloring of the status in the surveillance view.

### CODE REVIEW

**Comment by indigo423** [comment]
.

**Comment by fooker** [comment]
Dafuq? That's not the way reviews work...

File: opennms-doc/guide-user/src/asciidoc/text/surveillance-view.adoc [hunk 1]
```diff
@@ -24,6 +24,12 @@ At a glance, you want to know how many of my servers have an issue rather than h
 .Example of a configured _Surveillance View_
 image::surveillance-view/01_surveillance-view.png[]
 
+The visual indication for outages in the surveillance view cells is defined as the following:
+
+* No services down: green as normal
+* One (1) service down: yellow as warning
+* More than one (1) services down: red as critical
```
Comment (fooker) [inline opennms-doc/guide-user/src/asciidoc/text/surveillance-view.adoc]: Should this be (> 1) for "more than one"?

Comment (mvrueden) [inline opennms-doc/guide-user/src/asciidoc/text/surveillance-view.adoc]: :+1:

**Comment by mvrueden** [comment]
I agree with @fooker

**Pull request merged**